formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -117,44 +117,40 @@ let res =
 let res =
   (if true {false} else {false}) [@onEntireIf];
 
-let add a b => ((a [@onA]) + b) [@onEverything];
+let add a b => (+) (a [@onA]) b [@onEverything];
 
 let add a b =>
-  ((a [@onA]) + (b [@onB])) [@onEverything];
+  (+) (a [@onA]) (b [@onB]) [@onEverything];
 
-let add a b => a + (b [@onB]);
+let add a b => a + b [@onB];
 
 let both = (fun a => a) [@onEntireFunction];
 
 let both a b =>
-  ((a [@onA]) && b) [@onEverything];
+  (&&) (a [@onA]) b [@onEverything];
 
-let both a b => (a [@onA]) && (b [@onB] [@onB]);
+let both a b => a [@onA] && b [@onB] [@onB];
 
-let both a b => (a && b) [@onEverything];
+let both a b => (&&) a b [@onEverything];
 
 let thisVal = 10;
 
-let x = 20 + (
-  - (add thisVal thisVal [@onFunctionCall])
-);
-
 let x =
-  (20 + (- (add thisVal thisVal)))
-    [@onEverything];
+  20 + (- add thisVal thisVal [@onFunctionCall]);
 
 let x =
-  -
-  (add thisVal thisVal [@onFunctionCall]);
+  (+) 20 (- add thisVal thisVal) [@onEverything];
 
+let x = - add thisVal thisVal [@onFunctionCall];
 
 let x =
-  (- (add thisVal thisVal)) [@onEverything];
+  (~-) (add thisVal thisVal) [@onEverything];
```

Comment:
Since the unary `-` minus function application had an attribute on the entire application, it is converted to normal application syntax, and in the case of unary minus, the *actual* identifier is `~-`. It's just that when written as "prefix" application, its true nature is hidden away.